client/layout/index.jsx

```diff
@@ -155,7 +156,9 @@ const Layout = createReactClass( {
 				{ 'development' === process.env.NODE_ENV && (
 					<AsyncLoad require="components/webpack-build-monitor" placeholder={ null } />
 				) }
-				<InlineHelp />
+				{ /* TODO: Update InlineHelp to make sense in a Jetpack context */
```

Comment:
This todo could be marked with `@todo` annotation so that it pops up as todo in some IDEs.

If it makes sense, it could even be moved to an issue?

Reply:
👍 Removed the TODO and opened an issue instead, as this would require some more discussion and research: #26987.